management/pre_firebase.js

```diff
@@ -0,0 +1,76 @@
+const requestPromise = require('request-promise');
+
+// [START list_available_projects]
+async function listProjects() {
+  const accessToken = await getAccessToken();
+  const options = {
+    uri: 'https://firebase.googleapis.com/v1beta1/availableProjects',
+    method: 'GET',
+    headers: {
+      'Authorization': 'Bearer ' + '<OAUTH2_ACCESS_TOKEN>',
+    },
+    json: true,
+  };
+
+  try {
+    const resp = await requestPromise(options);
+    console.log(resp);
+    const projects = resp['projectInfo'];
+    console.log('Project total: ' + projects.length);
+    console.log('');
+    for (let i in projects) {
+      const project = projects[i];
+      console.log('Project ' + i);
+      console.log('ID: ' + project['project']);
+      console.log('Display Name: ' + project['displayName']);
+      console.log('');
+    }
+  } catch(err) {
+    console.error(err);
+  }
+}
+// [END list_available_projects]
+
+// [START add_firebase]
+async function addFirebase(projectId) {
+  const options = {
+    uri: 'https://firebase.googleapis.com/v1beta1/' + projectId + ':addFirebase',
+    method: 'POST',
+    headers: {
+      'Authorization': 'Bearer ' + '<OAUTH2_ACCESS_TOKEN>',
+    },
+    body: {
+      'timeZone': 'America/Los_Angeles',
+      'regionCode': 'US'
+    },
+    json: true,
+  };
+
+  try {
+    const resp = await requestPromise(options);
+    console.log(resp);
```

Comment:
Do you need to be doing `JSON.stringify(resp)` in all these snippets to make it actually useful?

Reply:
Indeed JSON.stringify would be better.